todo/src/scripts/sortDirect.js

```diff
@@ -0,0 +1,28 @@
+export default class {
+    constructor(){
+        this.init();
+    }
+
+    init() {
+        let direction = JSON.parse(localStorage.getItem('direction'));
+        if (direction === null) {
```

Comment:
`!direction` does this work

Reply:
Sorry, ```!import``` is not work, because direction can be equal ```true``` or ```false``` , and ```null``` if  ```localStorage``` is not contain ```direction``` value.